src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+import core.basesyntax.model.abstraction.Color;
+import core.basesyntax.model.abstraction.Figure;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle() {
+    }
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.ceil(Math.PI * radius * radius);
+    }
+
+    @Override
+    public String drawFigure() {
+        return "Figure: " + "circule" + ", area: " + getArea() + " sq.units, radius: "
+                + getRadius() + " units, color: " + getColor().toString().toLowerCase();
```

Comment:
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)
Use name() for getting String representation of enum constants

**This is for all classes - replace toString() with name()**